securedrop_export/export.py

```diff
@@ -134,27 +134,12 @@ def extract_tarball(self):
             self.exit_gracefully(msg)
 
     def check_usb_connected(self):
-
-        # If the USB is not attached via qvm-usb attach, lsusb will return empty string and a
-        # return code of 1
+        # If the USB is not attached via qvm-usb attach, lsblk will have a non-zero return code
         try:
-          p = subprocess.check_output(["lsusb", "-s", "{}:".format(self.pci_bus_id)])
+            subprocess.check_output(["lsblk", "-p", "-o", "KNAME", DEVICE], stderr=subprocess.PIPE)
```

Comment:
thanks @creviera this does indeed work reliably for disks!

The reason we used lsusb here is because we wanted to `check_usb_connected` function to be able to detect both printers and block devices.

The command here will return  32: `lsblk: /dev/sda1/ is not a block device`. I see 2 options:

a) search for a more reliable solution that works for printers and drives
b) we can separate the preflight checks for USB connected:
    i. For disks, using the method described here
    ii. For printers, using either lsusb or another method

what do you think?

Reply:
Oops, somehow I missed this comment! I seeeeeee. My vote goes to whatever stabilizes export the quickest, so I think that's b, and then we can research a.